test/Buyer/viewFarmOfferings.test.js

```diff
@@ -0,0 +1,15 @@
+import { describe } from "mocha";
+import nock from "nock";
+import { BASEURL } from "../../src/core/urls.js";
+
+describe("Buyer", () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/JoeMindo/USSDTest/pullRequest?prid=8695127)
```suggestion
describe('Buyer', () => {
```